### ISSUE TITLE
implement secure storage of media

### ISSUE THREAD

**Issue by n8fr8** [z3t0/CameraX#4]
using SQLCipher or IOCipher libraries

(see chatsecure iOS project for an example)

**Comment by z3t0** [comment]
Would this work? https://github.com/RNCryptor/RNCryptor
Otherwise there is KeyChain but that is only recommended for small amounts of data

**Comment by z3t0** [comment]
Or this may be more appropriate https://github.com/krzyzanowskim/CryptoSwift

**Comment by n8fr8** [comment]
Yeah, that looks good. I think whatever is easiest. Also encrypted storage isn't a high priority at first, but if it is easy to implement then great.
